VDF.Core/FFTools/FFProbeJsonReader.cs

```diff
@@ -18,114 +18,212 @@
 using System.Collections.Generic;
 using System.Text.Json;
 using VDF.Core.Utils;
+using System.Text.Json.Serialization;
+
+using DSO = System.Collections.Generic.Dictionary<string, object>; 
+using LO  = System.Collections.Generic.List <object>; 
 
 namespace VDF.Core.FFTools {
+
+#if true
```

Comment:
Remove this as well

Reply:
There are two versions of FFProbeJsonReader implementation.
Removing the ` #if ... #else ... #endif` would not work.

=> I offered you the two variants so you can choose which one you like better. I didn't know if there was a reason that you didn't use the ready deserialization function in your version.
And if there are still problems with one of the variants, you would still have the other to choose from.
Therefore, one would either delete one of the two variants completely, or save it just in case.